cicecore/cicedynB/general/ice_step_mod.F90

```diff
@@ -863,11 +864,12 @@ subroutine step_dyn_horiz (dt)
       call init_history_dyn     ! initialize dynamic history variables
 
       !-----------------------------------------------------------------
-      ! Elastic-viscous-plastic ice dynamics
+      ! Ice dynamics (momentum equation)
       !-----------------------------------------------------------------
 
       if (kdyn == 1) call evp (dt)
       if (kdyn == 2) call eap (dt)
+      if (kdyn == 3) call implicit_solver (dt)
```

Comment:
So do we use kdyn=3 in the cice6 configuration? Usually implicit solver is taking longer time, what is the impact on cice model running time when using kdyn=3?

Reply:
We run using evp (kdyn==1).